src/platform/nrfconnect/wifi/WiFiManager.h

```diff
@@ -155,31 +156,33 @@ class WiFiManager
     CHIP_ERROR GetNetworkStatistics(NetworkStatistics & stats) const;
 
 private:
-    using ConnectionParams = wifi_connect_req_params;
     using NetEventCallback = net_mgmt_event_callback;
     using WiFiStatus       = wifi_status;
-    using NetEventHandler  = void (*)(NetEventCallback *);
+    using NetInterfacePtr  = net_if *;
```

Comment:
Maybe it's just me, but IMHO aliasing Zephyr types isn't really useful as the code isn't generic enough to allow for easy swap of the underlying type, and it makes the code harder to follow, because I need to keep the mapping in mind to verify if we set all the fields properly. It makes sense in the case of callbacks, but if the original type name is self-describing, maybe "entities should not be multiplied beyond necessity"?
But let others comment on that, too :)

Reply:
Yehhh... It made more sense when I was using the `struct` here which was redundant.